breathe/directives/content_block.py

```diff
@@ -103,10 +103,8 @@ class DoxygenNamespaceDirective(_DoxygenContentBlockDirective):
 
 class DoxygenGroupDirective(_DoxygenContentBlockDirective):
     kind = "group"
-    option_spec = {
-        **_DoxygenContentBlockDirective.option_spec,
-        "inner": flag,
-    }
+    option_spec = _DoxygenContentBlockDirective.option_spec.copy()
+    option_spec.update({"inner": flag})
```

Comment:
Out of interest, why do you prefer this formulation? Does it have an objective benefit or just preferred style?

Reply:
It is just a style choice, to be consistent with the equivalent cases I have seen in Sphinx to more quickly spot what is going on.